### PR TITLE
STR-128 --FE: Create informational slide template

### PR DESCRIPTION
**Pull request by frlzjosh**
# Description
'Write a meaningful description of your task'
  Created a template that reflects the informational slides.
  Should include styling as well as the functionality
  Should be mobile responsive
  Our documents mention a carousel, do not worry about carousel.

# Testing
'What does the reviewer need to do to test your PR?'
 - Login as `april.feldman` 
 - Click on a module
 - You need to make sure that the info template looks like the mockup
 - Be as constructively critical as possible on my code and on my styling
 - Make sure that the user is only able to proceed after five seconds

### CODE REVIEW

File: public/css/app.css [hunk 1]
```diff
@@ -12049,16 +12049,28 @@ a.text-dark:focus {
 .module {
   overflow-x: hidden;
   height: 100vh;
+  margin-bottom: 4rem;
```
Comment (BrianLinggadjaja) [inline public/css/app.css]: This makes the whole page overflow with a 2nd scrollbar.

**Review by BrianLinggadjaja — DISMISSED** [review]
Everything looks good just one small change.

**Review by BrianLinggadjaja — APPROVED** [review]
Awesome!

**Review by andrewMETALAB — APPROVED** [review]
great work 👍

**Pull request merged**